src/common/addlayers/AddLayersDirective.js

```diff
@@ -27,7 +27,7 @@
 
             // default to the Local Geoserver. Note that when a map is saved and loaded again,
             // the order of the servers might be different and MapLoom should be able to handle it accordingly
-            var server = serverService.getServerLocalGeoserver();
+            var server = serverService.getElasticLayerConfig();
```

Comment:
Would it make sense to change the method to getConfig() or is it specific elastic configuration?